pkg/backup/backup/backup_manager.go

```diff
@@ -65,7 +66,26 @@ func (bm *backupManager) syncBackupJob(backup *v1alpha1.Backup) error {
 	name := backup.GetName()
 	backupJobName := backup.GetBackupJobName()
 
-	err := backuputil.ValidateBackup(backup)
+	var err error
+	if backup.Spec.BR == nil {
```

Comment:
@DanielZhangQD you commented you check if `backup.Spec.From` is nil in the `cmd/backup-manager/app/export/manager.go`, however, we validated the `backup.Spec.From` here and make sure  it exists. Then do we still need the check?